src/drivers/mct/main/cime_comp_mod.F90

```diff
@@ -3573,7 +3587,7 @@ subroutine cime_run()
          call seq_rest_write(EClock_d, seq_SyncClock, infodata,       &
               atm, lnd, ice, ocn, rof, glc, wav, esp,                 &
               fractions_ax, fractions_lx, fractions_ix, fractions_ox, &
-              fractions_rx, fractions_gx, fractions_wx)
+              fractions_rx, fractions_gx, fractions_wx, tag=trim(cpl_inst_tag))
```

Comment:
Why specify the name of a non-optional input? Seems inconsistent.

Reply:
fixed